Package.swift

```diff
@@ -19,6 +19,7 @@ let package = Package(
         .package(url: "https://github.com/jpsim/SourceKitten.git", from: "0.31.1"),
         .package(url: "https://github.com/jpsim/Yams.git", from: "4.0.2"),
         .package(url: "https://github.com/scottrhoyt/SwiftyTextTable.git", from: "0.9.0"),
+        .package(url: "https://github.com/apple/swift-markdown.git", .branch("main")),
```

Comment:
We can't depend on branches here for published packages. Is there a version we can pin to?